tests/test_datasets/test_dataset_functions.py

```diff
@@ -27,6 +27,7 @@
 from openml.datasets.functions import (
     create_dataset,
     edit_dataset,
+    fork_dataset,
```

Comment:
Could you please also add the function to the documentation and also to https://github.com/openml/openml-python/blob/develop/openml/datasets/__init__.py and maybe not import from functions.py as done here (I know you didn't write the original code, but we should try to get rid of importing from functions.py if not necessary).